tdigest/merging_digest.go

```diff
@@ -481,3 +482,32 @@ func (td *MergingDigest) Data() *MergingDigestData {
 		ReciprocalSum: td.reciprocalSum,
 	}
 }
+
+// decay decays the histo to make values at the top less interesting over time
+func (td *MergingDigest) decay(p float64) {
+	td.mergeAllTemps()
+	for _, c := range td.mainCentroids {
+		c.Weight = c.Weight * p
+		c.Mean = c.Mean * p
+	}
```

Comment:
This does what it says, but I have no idea if this is mathematically valid. :)

Reply:
From the chief data scientist.

Reply:
Ack!